### PR TITLE
chore(docs): fix authenticator page for vue 

### PR DESCRIPTION
**Pull request by esauerbo**
#### Description of changes
Fixes a few typos on docs site 

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by dbanksdesign — DISMISSED** [review]

File: docs/src/data/links.tsx [hunk 1]
```diff
@@ -619,7 +619,7 @@ export const gettingStarted: ComponentNavItem[] = [
   {
     href: '/getting-started/migration',
     label: 'Migration',
-    platforms: ['react', 'vue', 'angular'],
+    platforms: ['react', 'vue', 'angular', 'react-native'],
```
Comment (reesscot) [inline docs/src/data/links.tsx:622]: oops, were we hiding the migration page for react native?

File: docs/src/pages/[platform]/connected-components/authenticator/quick-start-add.web.mdx [hunk 2]
```diff
@@ -148,7 +148,7 @@ export class UseAuthenticatorComponent {
   </Alert>
 </InlineFilter>
 <InlineFilter filters={['vue']}>
-  For Vue 3, import the `Authenticator` and the `styles.css` into your single file component. You can then use the `<authenticator>` inside your template.
+  For Vue 4, import the `Authenticator` and the `styles.css` into your single file component. You can then use the `<authenticator>` inside your template.
```
[the remark this answers is not in the document]
Comment (reesscot) [reply docs/src/pages/[platform]/connected-components/authenticator/quick-start-add.web.mdx:151]: +1, have we tested in Vue 4?

File: docs/src/pages/[platform]/getting-started/migration/migration.angular.mdx [hunk 3]
```diff
@@ -126,7 +126,7 @@ The input and return type interfaces are available as imports from `aws-amplify/
 import { ConfirmSignInInput } from 'aws-amplify';
 ```
 
-##$ 3.x to 4.x
+## 3.x to 4.x
```
Comment (reesscot) [inline docs/src/pages/[platform]/getting-started/migration/migration.angular.mdx:129]: good catch

File: docs/src/data/links.tsx (hunk 1 above)
Comment (esauerbo) [reply docs/src/data/links.tsx:622]: yeah :(

File: docs/src/pages/[platform]/connected-components/authenticator/quick-start-add.web.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (esauerbo) [reply docs/src/pages/[platform]/connected-components/authenticator/quick-start-add.web.mdx]: I totally thought this was talking about the ui-vue version, changed to 3

**Review by hbuchel — APPROVED** [review]

**Review by reesscot — APPROVED** [review]

**Pull request merged**